packages/administration/src/Controller/AbstractCrudControllerExtension.php

```
@@ -29,4 +30,8 @@ public function configureDatagrid(Datagrid $datagrid): void
     public function configureQuery(QueryBuilder $queryBuilder): void
     {
     }
+
+    public function configureForm(CrudFormConfigurator $formConfigurator, ?object $entity = null): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$formConfigurator'. (undefined)

(UnusedFormalParameter)